### DISCUSSION TITLE
docker-compose (dockge) without a proxy

### DISCUSSION THREAD

**Discussion by Solear** [seanmorley15/AdventureLog#349] [Q&A] [answered]
I'm to stupid to fire this composes up.

Is there a simple way without a proxy (nginx or traefic)? I dont need a proxy (access only via wireguard). 
I would like to use dockge (in a proxmox LXC) and a simple docker-compose.
Thank you! :-)

**Comment by seanmorley15** [comment, marked as answer]
Hi @Solear, the most recent update has removed the need for a separate proxy. Take a look at the [docs](https://docs.adventurelog.app/docs/Installation/docker) to see the new docker install instructions if you want a docker install. There is also a Proxmox LXC script that can be found [here](https://community-scripts.github.io/ProxmoxVE/scripts?id=adventurelog).
Hope this helps!
